u8timeseries/custom_logging.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import logging
+from datetime import datetime
+import pathlib
+import time
+
+def get_logger(name):
+    """
+    Internally calls the logging.getLogger function with the 'name' argument to create or 
+    retrieve a logger object. It is recommended to pass __name__ as argument when calling 
+    get_logger. The returned logger object logs to the standard error stream and formats
+    the messages appropriately.
+
+    :param name: The name that gets passed to the logger.getLogger function.
+    :return: A logger instance with the given name.
+    """
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.INFO)
+    stderr_handler = logging.StreamHandler()
+    formatter = logging.Formatter('[%(asctime)s] %(levelname)s | %(name)s | %(message)s')
+    stderr_handler.setFormatter(formatter)
+    logger.addHandler(stderr_handler)
+    return logger
+
+def check_value_log(boolean_value: bool, message: str = "", logger: logging.Logger = get_logger('main_logger')):
```

Comment:
I'm nitpicking a bit, but this is used very often. I think `check_value_log()` doesn't really reflect the fact it's checking a boolean predicate and possible.

How about naming it `raise_if_not()`or something like this? @TheMP any idea?